contracts/dtoken/src/views.rs

```diff
@@ -0,0 +1,110 @@
+use crate::*;
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize, Serialize, Deserialize)]
+#[serde(crate = "near_sdk::serde")]
+#[derive(Debug)]
+pub struct MarketData {
+    pub market_total_supplies: Balance,
```

Comment:
@pahomas it seems that `total_supplies` is enough (instead of `market_total_supplies`) as it reads 
MarketData.total_supplies